stix_shifter_modules/azure_sentinel/stix_translation/json/alert_from_stix_map.json

```diff
@@ -169,7 +109,62 @@
       "dst_geolocation": ["networkConnections.destinationLocation"],
       "src_application_user_ref.user_id":["userStates.aadUserId"],
       "src_application_user_ref.type":["userStates.logonType"],
-      "time_observed": ["lastModifiedDateTime"]
+      "time_observed": ["lastModifiedDateTime"],
+      "x_activityGroupName": ["activityGroupName"],
+      "x_assignedTo": ["assignedTo"],
+      "x_comments": ["comments"],
+      "x_confidence": ["confidence"],
+      "x_detectionIds": ["detectionIds"],
+      "x_feedback": ["feedback"],
+      "x_id": ["id"],
+      "x_incidentIds": ["incidentIds"],
+      "x_recommendedActions": ["recommendedActions"],
+      "x_sourceMaterials": ["sourceMaterials"],
+      "x_status": ["status"],
+      "x_tags": ["tags"],
+      "x_cloudAppStates.destinationServiceName": ["cloudAppStates.destinationServiceName"],
+      "x_cloudAppStates.destinationServiceIp": ["cloudAppStates.destinationServiceIp"],
+      "x_cloudAppStates.riskScore": ["cloudAppStates.riskScore"],
+      "x_hostStates.isAzureAadJoined": ["hostStates.isAzureAadJoined"],
+      "x_hostStates.isAzureAadRegistered": ["hostStates.isAzureAadRegistered"],
+      "x_hostStates.isHybridAzureDomainJoined": ["hostStates.isHybridAzureDomainJoined"],
+      "x_hostStates.os": ["hostStates.os"],
+      "x_hostStates.publicIpAddress": ["hostStates.publicIpAddress"],
+      "x_hostStates.privateIpAddress": ["hostStates.privateIpAddress"],
+      "x_hostStates.riskScore": ["hostStates.riskScore"],
+      "x_malwareStates.category": ["malwareStates.category"],
+      "x_malwareStates.family": ["malwareStates.family"],
+      "x_malwareStates.name": ["malwareStates.family"],
+      "x_malwareStates.severity": ["malwareStates.family"],
+      "x_malwareStates.wasRunning": ["malwareStates.family"],
+      "x_networkConnections.applicationName": ["networkConnections.applicationName"],
+      "x_networkConnections.direction": ["networkConnections.direction"],
+      "x_networkConnections.domainRegisteredDateTime": ["networkConnections.domainRegisteredDateTime"],
+      "x_networkConnections.localDnsName": ["networkConnections.localDnsName"],
+      "x_networkConnections.natDestinationPort": ["networkConnections.natDestinationPort"],
+      "x_networkConnections.natSourcePort": ["networkConnections.natSourcePort"],
+      "x_networkConnections.riskScore": ["networkConnections.riskScore"],
+      "x_networkConnections.status": ["networkConnections.status"],
+      "x_processes.integrityLevel": ["processes.integrityLevel"],
+      "x_processes.isElevated": ["processes.isElevated"],
+      "x_securityResources.resource": ["securityResources.resource"],
+      "x_securityResources.resourceType": ["securityResources.resourceType"],
+      "x_triggers.name": ["triggers.name"],
+      "x_triggers.type": ["triggers.type"],
+      "x_triggers.value": ["triggers.value"],
+      "x_userStates.logonIp": ["userStates.logonIp"],
```

Comment:
Could all the userStates mappings go under `user-account` if they are specific to a user?

Reply:
yes. done